CHANGELOG.md

```diff
@@ -41,6 +41,8 @@
 - Improved general testing infrastructure
 - Added test cases for the index function in `Controllers/Course`
 - Added test cases for the depts function in `Controllers/Course`
+- Added test cases for the courseInfo function in `Contollers/Course`
+- Added a helper function for testing  the courseInfo function in `Controllers/Course` in `backend-test/TestHelpers.hs` to simulate a query for courseInfo
```

Comment:
While this is indeed a change you made, since it was so tied to the previous entry, I don't think it's worth including here.